core/sql/regress/executor/EXPECTED130

```diff
@@ -124,7 +134,16 @@ inserted row13
 
 --- 3 row(s) selected.
 >>
->>
+>>values (filetolob('./myfile.txt'));
+
+*** ERROR[8442] Unable to access ExpLOB interface. Call to ExpLOB returned error LOB_PTR_ERROR(532). Error detail 0.
```

Comment:
So, the problem in this case is that ./myfile.txt simply doesn't exist? Would it be better to simply say that? The error message as written seems to suggest some sort of internal error which will likely lead to a support call. A customer won't have any idea what an ExpLOB interface is, so we shouldn't tell him about it unless we expect him to do something with it.

I'm wondering if there are two cases here. One might be the user specified a file name that doesn't exist (as in this example). Another might be that Trafodion stored a file name internally and that file was then deleted. That would be an internal error if Trafodion managed the file I suppose. But if it is an external file, then it is a user error. Perhaps we should simply report that the file doesn't exist.